effectiveOfficeBackend/src/main/kotlin/office/effective/common/notifications/FcmNotificationSender.kt

```diff
@@ -29,6 +45,58 @@ class FcmNotificationSender(private val fcm: FirebaseMessaging): INotificationSe
             .build()
         fcm.send(msg)
     }
+    
+    override fun sendUpdateContentMessages(topic: String, resourceId: String) {
+        logger.info("[fcmNotificationSender] Received update on {} calendar id", resourceId)
+        val workspace = calendarIdsRepository.findWorkspaceById(resourceId)
+        if (workspace.tag != "meeting") return
+        
+        val startTime = ZonedDateTime
+            .now(ZoneId.of(BookingConstants.DEFAULT_TIMEZONE_ID))
+            .toEpochSecond() * 1000
+        val endTime = startTime + ONE_WEEK_IN_MILLISECONDS
+        
+        val bookings = bookingService.findAll(
+            workspaceId = workspace.id,
+            bookingRangeFrom = startTime,
+            bookingRangeTo = endTime
+        )
+        
+        if (bookings.isEmpty()) {
+            logger.info("[fcmNotificationSender] Bookings on 1 week from now for {} workspace were empty", workspace.name)
```

Comment:
The name of the class is automatically specified by the log. The name of the method should be specified at the beginning of the message: [sendUpdateContentMessages]